src/modules/sector-industries/dto/get-sector-industries-query.dto.ts

```diff
@@ -6,13 +6,13 @@ export class GetSectorIndustriesQueryDto {
   @IsString()
   @MaxLength(4)
   @Matches(/^\d{4}$/)
-  @ApiProperty({ example: '1010' })
+  @ApiProperty({ required: false, example: '1010', description: 'Search by UKEF Sector id, returns multiple Industries' })
   public ukefSectorId: string;
 
   @IsOptional()
   @IsString()
   @MaxLength(5)
```

Comment:
Can we also add `@Min(2)`, or a number you think is a common min standard please?

Reply:
Changed MaxLength to Length, this tests min and max.

Length validation duplicates regex check, regex is more precise because just digits are allowed.